packages/generator/test/test-util/data-model.ts

```diff
@@ -10,6 +10,9 @@ import {
   VdmUnsupportedReason
 } from '../../src/vdm-types';
 
+/**
+ * @internal
+ */
```

Comment:
[info] Same as above, don't forget to remove these comments and all other comments in the test files if tests are excluded.